www/docs/server/adapters/fetch.mdx

```diff
@@ -274,6 +274,79 @@ Run `wrangler dev server.ts` and your endpoints will be available via HTTP!
 | `getUser`    | `GET http://localhost:8787/trpc/getUserById?input=INPUT` <br/><br/>where `INPUT` is a URI-encoded JSON string. |
 | `createUser` | `POST http://localhost:8787/trpc/createUser` <br/><br/>with `req.body` of type `User`                          |
 
+### Deno Oak
+
+#### Install dependencies
+
+<Tabs>
+  <TabItem value="macOS / Linux" label="macOS / Linux" default>
+
+```sh
+curl -fsSL https://deno.land/x/install/install.sh | sh
+```
+
+  </TabItem>
+  <TabItem value="Windows" label="Windows">
+
+```sh
+irm https://deno.land/install.ps1 | iex
+```
+
+  </TabItem>
+</Tabs>
```

Comment:
should this be part of our installation guide?

seems fairly obvious you need Deno? We don't tell you to install Node for the otehr examples

Reply:
Agreed - I only added it because it was under Deno Deploy. I suppose I should remove the installation commands from both of them?

Reply:
yep that sounds good to me